graphiti_core/storage/kuzu_driver.py

```
@@ -0,0 +1,57 @@
+        
+        Returns:
+            A tuple of (rows, None, column_names), where rows is a list of dictionaries mapping column names to values, and column_names is a list of column names.
+        """
+        parameters.pop('database_', None)
+        parameters.pop('routing_', None)
+        result = await self.conn.execute(query, parameters or None)
+        cols = result.get_column_names()
+        rows = []
+        while result.has_next():
+            row = result.get_next()
+            rows.append({c: v for c, v in zip(cols, row, strict=False)})
```

Pylint (3.3.7)

[refactor] Unnecessary use of a comprehension, use dict(zip(cols, row, strict=False)) instead.

(R1721)